packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -253,6 +279,30 @@ const ContractDetails = ({
                     label={localize('Start time')}
                     value={toGMTFormat(epochToMoment(Number(date_start))) || ' - '}
                 />
+                {isLookBacksContract(contract_type) && (
+                    <React.Fragment>
+                        {contract_type === CONTRACT_TYPES.LB_HIGH_LOW ? (
+                            <React.Fragment>
+                                {[high_barrier, low_barrier].map((barrier, index) => (
+                                    <ContractAuditItem
+                                        id={`dt_bt_label_${index + 1}`}
+                                        icon={createLookBacksMarker(index === 0 ? 'H' : 'L').icon}
```

Comment:
Can we save the response of `createLookBacksMarker(index === 0 ? 'H' : 'L') `  in a variable and use it just so that we don't have to use conditional operator again .

Reply:
Good idea, will apply